packages/dx-react-scheduler-material-ui/src/templates/views/common/time-table/layout-container.jsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import RootRef from '@material-ui/core/RootRef';
+
+export class TimeTableContainer extends React.PureComponent {
+  constructor(props) {
+    super(props);
+
+    this.table = null;
+
+    this.setCells = this.setCells.bind(this);
+    this.saveReference = this.saveReference.bind(this);
```

Comment:
Why not [field declaration](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes#Field_declarations)?

Reply:
What do you mean?

Reply:
```jsx
class TimeTableContainer ... {
  ...
  saveReference = (ref) => {
    const { tableRef } = this.props;
    this.table = ref;
    tableRef(ref);
  };
  ...
}
```

Reply:
As I know, it works in TS only

Reply:
Yes, I forgot that it is another package.
I withdraw the question.